integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -68,13 +72,16 @@ import com.okta.sdk.tests.it.util.ITSupport
 import org.testng.Assert
 import org.testng.annotations.Test
 
+import static com.okta.sdk.tests.it.util.Util.assertPresent
+import static com.okta.sdk.tests.it.util.Util.assertNotPresent
+
 import static org.hamcrest.MatcherAssert.assertThat
 import static org.hamcrest.Matchers.equalTo
 import static org.hamcrest.Matchers.notNullValue
 import static org.hamcrest.Matchers.sameInstance
 
 /**
- * Tests for /api/v1/apps.
+ * Tests for /api/v1/apps
```

Comment:
```suggestion
 * Tests for {@code /api/v1/apps}.
```
This was _probably_ poor javadoc to start with but the period. The first sentence _should_ end in a period.